resources/user-guides/attach-data.md

```diff
@@ -233,7 +234,7 @@ In the following example, an mbox named **custom_mbox** with an additional mbox
 
 ![](../../.gitbook/assets/target-attach-data-json-example-prefetch.png)
 
-In the above example, the JSON payload adds custom mbox parameters only for the custom mbox added. Custom parameters can be added to each of the Target objects instead. The following example contains a valid JSON payload for this use case:
+In the above example, the JSON payload adds custom mbox parameters only for the custom mbox added. Custom parameters can be added to each of the Target prefetch objects instead. The following example contains a valid JSON payload for this use case:
```

Comment:
Please reword as follows:
"In the above example, the JSON payload adds custom mbox parameters only for the custom mbox that was added. You can also add custom parameters to each of the Target retrieve prefetch objects. The following example contains a valid JSON payload for this use case:"

Reply:
I've updated to this suggestion but left out the word `retrieve` as it doesn't apply to prefetch.